app/templates/snips/studentHistoryModal.html

```diff
@@ -7,40 +7,40 @@
   <button type="button" class="close" data-dismiss="modal">&times;</button>
 
   <span style="text-align: right;"><button type="button" class="btn btn-default btn col-sm-2" id="print" onclick="fillPDF({{statusForm.laborStatusFormID}})"><span class="glyphicon glyphicon-print"></span> Print</button></span>
-  <span class="h3 col-sm-9 modalHeader">
+  <span class="h3 col-sm-9 modalHeader" id="studentDetails">
       {{statusForm.studentSupervisee.FIRST_NAME}} {{statusForm.studentSupervisee.LAST_NAME}} <a href="mailto:{{statusForm.studentSupervisee.STU_EMAIL}}"><span class="glyphicon glyphicon-envelope mailtoIcon"></span></a>
       <br><span class="h4">{{statusForm.studentSupervisee.ID}}</span>
       {% if statusForm.studentSupervisee.FIRST_NAME + " " + statusForm.studentSupervisee.LAST_NAME != statusForm.studentName %}
       <div class="original-name">Originally filed under "{{statusForm.studentName}}"</div>
       {% endif %}
   </span>
-
+  <div class="btn btn-primary" id="easyEmail" onClick="mailPositionToLabor()"><span>Email Labor</span></div>
 </div>
 <div class="modal-body" id="modalid">
```

Comment:
This id makes you all look like saints in comparison and our lord and Savior Brian is upset at this blasphemy. Please fix the great mistakes of your ancestors and change this id to please the gods.